content/en/marketplace/marketplace.md

```diff
@@ -0,0 +1,192 @@
+2. Log into the [Datadog Partner Portal](https://partners.datadoghq.com/English/)
+3. On your personal homepage, click on the “Learn More” button under “Sandbox Access”
+4. Select “Request Sandbox Upgrade”
+
+Converting an account to a developer sandbox may take up to one or two business days. Please contact [Datadog Support](https://docs.datadoghq.com/help/) with any questions.
+
+
+## Develop your Integration
+
+### Resources
+
+In addition to this documentation, you can learn more about developing Datadog integrations by taking the following steps :
+
+* Attend our on-demand [“Introduction to Datadog Integrations”](https://learn.datadoghq.com/course/view.php?id=38) course on the [Datadog Learning Center](https://learn.datadoghq.com/).
+* Review our example pull-request in the [Marketplace repository](https://github.com/DataDog/marketplace) with annotations and best practices (this link is only available to Technology Partners that have have completed the Marketplace Listing Agreement in the Datadog Partner Portal)
+* Explore existing integrations built by other Technology Partners in our [Integrations Extras repository](https://github.com/DataDog/integrations-extras)
+* Join our Marketplace Engineering Office Hours through the [Datadog public Slack](https://chat.datadoghq.com/)
+
+### The Development Process
+
+1. [Choose the type of Marketplace offering you would like to list](#1-choose-an-integration-type)
+2. [Build a bi-directional integration](#2-build-a-bi-directional-integration)
+3. [Clone either the Marketplace or the Integrations Extras repository](#3-clone-either-the-marketplace-or-the-integrations-extras-repository)
+4. [Install the Datadog Development Toolkit](#4-install-and-run-the-datadog-development-toolkit)
+5. [Populate the tile scaffolding with metadata and out-of-the-box assets (like  dashboards and monitors)](#5-populate-the-tile-scaffolding)
+6. [Submit a pull-request](#6-submit-a-pull-request)
+7. [Approve tile for release](#7-approve-tile-for-release)
+
+### 1. Choose an integration type
+
+There are a few different ways to integrate with Datadog. Depending on the use case and type of integration, choose the approach that makes the most sense for your offering. 
+
+#### [**1. Datadog Agent-based Integration**](https://docs.datadoghq.com/developers/integrations/)
+
+##### [OpenMetrics check](https://docs.datadoghq.com/developers/prometheus/)
+* A OpenMetrics check is suitable for gathering telemetry data from existing applications that expose metrics using the Open Metrics standard.
+
+##### [Python Check](https://docs.datadoghq.com/developers/integrations/new_check_howto/?tab=configurationtemplate#write-the-check)
+* A Python check is suitable for monitoring services or products that do not expose metrics in a standard format. The Python check is used to collect telemetry data from various APIs or command line tools.
+
+##### [DogStatsD](https://docs.datadoghq.com/developers/dogstatsd/?tab=hostagent)
```

Comment:
I'm not sure I see a use case for partners building a Dogstatsd integration, should we remove this section?

Reply:
Leaving this in for now, since the agent team said they aren't sure if there are DogStatsD integration today, and this is already included in the current tech partner guide. We can remove it later based on feedback.